astropy/table/operations.py

```diff
@@ -129,6 +130,86 @@ def join(left, right, keys=None, join_type='inner',
     return out
 
 
+def setdiff(table1, table2, keys=None):
+    """
+    Take a set difference of table rows.
+
+    The row set difference will contain all rows in table1 that are not
+    present in table2.
+
+    Parameters
+    ----------
+    table1 : Table object
+        table1 is on the left side of the set difference.
+    table2 : Table object
+        table2 is on the right side of the set difference.
+    keys : str or list of str
+        Name(s) of column(s) used to match rows of left and right tables.
+        Default is to use all columns which are common to both tables.
```

Comment:
Whoops, I copy pasted that from the join doc string and didn't read it all the way through 
>.<